src/couch/src/couch_compress.erl

```diff
@@ -24,10 +24,11 @@
 %      http://www.erlang.org/doc/apps/erts/erl_ext_dist.html
 -define(TERM_PREFIX, 131).
 -define(COMPRESSED_TERM_PREFIX, 131, 80).
+-define(DEFAULT_FILE_COMPRESSION, "snappy").
 
 
 get_compression_method() ->
-    case config:get("couchdb", "file_compression") of
+    case config:get("couchdb", "file_compression", ?DEFAULT_FILE_COMPRESSION) of
```

Comment:
I think we can re-use the constant from couch_db.hrl:

https://github.com/apache/couchdb/blob/3.x/src/couch/include/couch_db.hrl#L16

Reply:
Changed back.